### PR TITLE
feat(Structures): add missing Guild properties

### PR DESCRIPTION
**Pull request by SiroTM**
This PR adds two properties:  `stage_instances` and `threads`. It also sets the type of `emojis`.

### CODE REVIEW

File: structures/Guild.ts [hunk 1]
```diff
@@ -40,8 +41,10 @@ export default interface Guild extends Base {
   roles: unknown[]; // TODO roles
   rules_channel_id: string | null;
   splash: string | null;
+  stage_instances?: unknown[]; // TODO stage instance
```
Comment (bsian03) [inline structures/Guild.ts]: Stage instance is done

**Review by bsian03 — CHANGES_REQUESTED** [review]

**Review by Khaaz — APPROVED** [review]
lgtm

File: structures/Guild.ts [hunk 2]
```diff
@@ -40,8 +42,10 @@ export default interface Guild extends Base {
   roles: unknown[]; // TODO roles
   rules_channel_id: string | null;
   splash: string | null;
+  stage_instances?: StageInstance[];
   system_channel_flags: SystemChannelFlags;
   system_channel_id: string | null;
+  threads?: Channel[];
```
Comment (Khaaz) [inline structures/Guild.ts:48]: threads are regular channels?

File: structures/Guild.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import Base from './Base';
 import Channel from './Channel';
 import GuildMember from './GuildMember';
+import Emoji from './Emoji';
```
Comment (bsian03) [inline structures/Guild.ts]:
```suggestion
import Emoji from './Emoji';
import GuildMember from './GuildMember';
```
OCD, sorry

**Review by bsian03 — APPROVED** [review]

**Pull request merged**